Source/UETorch/Private/TorchContext.cpp

```diff
@@ -11,6 +11,8 @@
 #include "ScriptBlueprintGeneratedClass.h"
 #include "TorchContext.h"
 
+DEFINE_LOG_CATEGORY(LogUETorch);
```

Comment:
I'm not sure you want to do this right now, because some related calls like `ScriptPlugin::CallFunction()` a=use LogScriptPlugin. Maybe it's better to have a single key?